### PR TITLE
Fix CSS whitescreen

### PR DESCRIPTION
**Pull request by glennreyes**
The app didn't display anything, this PR adds some CSS to fix it. Also formats relevant files with prettier.

image

How can one work like that? 😂

### CODE REVIEW

**Comment by nikgraf** [comment]
@glennreyes How can I reproduce the issue? When did it happen?

**Comment by glennreyes** [comment]
It occurs on local dev with proxy using Chrome. Any page look like this. The 2 lines of css in the file upload container did the fix.

File: src/containers/file-upload/FileUploadProvider.js [hunk 1]
```diff
@@ -22,7 +22,9 @@ export default class FileUploadProvider extends PureComponent {
   render() {
     return (
       <Provider store={getStore()}>
-        <ConnectedFileUpload>{this.props.children}</ConnectedFileUpload>
+        <ConnectedFileUpload {...this.props}>
```
Comment (nikgraf) [inline src/containers/file-upload/FileUploadProvider.js]: Why this change?

File: src/components/file-upload/FileUpload.js [hunk 2]
```diff
@@ -63,26 +63,28 @@ export default class FileUpload extends PureComponent {
...
-        disabled={disabled}
-        style={dropZoneStyle}
         onDrop={this.onDragDone}
         onDragStart={this.onDragStart}
         onDragEnter={this.onDragEnter}
         onDragLeave={this.onDragDone}
         onDropAccepted={this.onAccept}
         onDropRejected={this.onReject}
         maxSize={maxSize}
+        {...rest}
```
Comment (nikgraf) [inline src/components/file-upload/FileUpload.js]: Let's try to use explicit props instead of spreading. It quickly becomes a trap for unexpected behaviour.

File: src/components/app-layout/AppLayout.js [hunk 3]
```diff
@@ -23,6 +23,10 @@ const Noop = () => null
     flexShrink: 0,
     flexDirection: 'column',
   },
+  fileUpload: {
```
Comment (nikgraf) [inline src/components/app-layout/AppLayout.js]: I wonder why this happening on your machine. Doesn't happen to nor did we have bug repots, hmmmm

Comment (nikgraf) [reply src/components/app-layout/AppLayout.js]: Wow, after switching to yarn now I have it too 💥

File: src/components/file-upload/FileUpload.js (hunk 2 above)
Comment (glennreyes) [reply src/components/file-upload/FileUpload.js]: Yep sounds good! I think `style`, `disabled` & `className` is what we would need here.

File: src/components/file-upload/FileUpload.js [hunk 4]
```diff
@@ -63,26 +63,28 @@ export default class FileUpload extends PureComponent {
     const {
       disabled,
       children,
-      dropZoneStyle,
       uploads,
       onNotify,
+      onReject,
+      onUpload,
```
Comment (glennreyes) [inline src/components/file-upload/FileUpload.js]: `onReject`, `onUpload` can be removed after https://github.com/ubergrape/grape-web-client/pull/722/files#r200601332

File: src/containers/file-upload/FileUploadProvider.js (hunk 1 above)
Comment (glennreyes) [reply src/containers/file-upload/FileUploadProvider.js]: For simple passing the props to the actual component.
We can pass explicit props instead, then `className` is the one we would need in this PR.
